app/frontend/components/domains/requirements-library/requirements-block-modal/fields-setup.tsx

```diff
@@ -258,6 +264,22 @@ export const FieldsSetup = observer(function FieldsSetup({
               {t(isInReorderMode ? "ui.done" : "ui.reorder")}
             </Button>
           </Flex>
+          {errors.displayName && (
```

Comment:
consider showing this error for any of the keys within errors. Might as well right?